edith-step/src/App.js

```diff
@@ -1,46 +1,102 @@
 import React, {Component} from 'react';
-import ReceiptInput from './ReceiptInput';
-import LineChart, {BarChart} from './UserChart';
-import axios from 'axios';
-//import "./App.css";
+import LineChart, {BarGraph, CategoryDoughnutChart, ItemDoughnutChart} from './UserChart';
+import ReceiptInput from './components/ReceiptInput';
+import TopNavbar from './components/TopNavbar';
+import './App.css';
 
+/**
+ * Corresponds to the different chart types.
+ */
+ const chart = {
+    'LINE': 'LINE',
+    'BAR': 'BAR',
+    'DOUGHNUT':  'DOUGHNUT' 
+}
 
-/** Main webpage for the website. */
+/**
+ * Main webpage for the website.
+ */
```

Comment:
I'm guessing this was the auto-formatting- which I'm cool with, just doesn't entirely make sense to me lol. I feel the single line comment was cleaner tbh

Reply:
I changed it.